server/http_config_device_handler.go

```diff
@@ -114,12 +114,14 @@ func deviceConfigMap[T any](class templates.Class, dev config.Device[T]) (map[st
 	if dc["config"] == nil {
```

Comment:
In dieses If laufen wir im Fall das Custom Device (Zeile 110) gar nicht rein. Der vorherige Code hätte dann meiner Erinnerung nach bereits das Richtige getan.